lib/menu.rb

```diff
@@ -0,0 +1,40 @@
+# Menu class
+# initiated with empty list of available dishes
+# can add dish to the list of available dishes
+# can be asked to view its list of available dishes
+# can be asked to select a dish from list and add it to basket
+
+require_relative './dish'
+require_relative './basket'
+
+class Menu
+  attr_reader :available_dishes, :dish_class, :basket
+  
+  def initialize(dish_class = Dish, basket = Basket.new)
+    @available_dishes = []
+    @dish_class = dish_class
+    @basket = basket
+  end
+
+  def new_dish(name, price)
+    @available_dishes << @dish_class.new(name, price)
+  end
```

Comment:
I was a little confused about this method as the user story is from the customer point of view. This is to add new dishes from the restaurant owner point of view?